Quick-Start.md

```diff
@@ -0,0 +1,209 @@
+
+```text
+Total:
+    CPU: 1600m (1.6 vCPU)
+    Memory: 2048Mi (2 GiB)
+```
+
+> [!NOTE]
+> The above resources are just for the infrastructure pods. To be able to run actual
+> workloads on your cluster you will need additional resources based on the size
+> and type of workload.
+
+### GPU Worker Node
+
+> [!IMPORTANT]
+> This step is necessary only if you require GPU capabilities for your workloads and your OpenShift cluster does not already include GPU-equipped nodes, follow these steps:
+
+1. **Open the OpenShift Cluster Console.**
+2. Navigate to **your-cluster** -> **Machine pools**.
+3. Click on **“Add machine pool”**.
+4. Provide a **name** for the new machine pool.
+5. In the **“Compute node instance type”** dropdown, scroll all the way down and search for the GPU instance type `g4dn.xlarge` or similar.
+6. Click on **Add machine pool** to finalize the creation of your new GPU-enabled machine pool.
+
+After adding the machine pool, OpenShift will begin provisioning the new GPU worker
+node. This process can take a few minutes. Once completed, the new node will be ready to handle GPU-accelerated workloads.
+
+> [!NOTE]
+> The `g4dn.xlarge` instance type is used for GPU worker nodes. Ensure this instance type meets your application needs or select another as required.
+
+## Automatic deployment for developers (optional - skip to run steps manually)
+
+As a quick alternative to the following manual deployment steps an automatic _Makefile_ script can be used to deploy the CodeFlare stack. This script also deploys the prerequisite operators and the entire CodeFlare stack.
+
+1. Clone the Repository
+
+```bash
+git clone https://github.com/project-codeflare/codeflare-operator.git
+cd codeflare-operator
+```
+
+2. Run the Makefile script
+
+```bash
+make all-in-one
+```
+
+> [!TIP]
+> Execute `make help` to list additional available operations.
+
+### Next Steps After Automatic Deployment
+
+After the automatic deployment is complete, you can proceed directly to the section [Configure Kueue for Task Scheduling](#configure-kueue-for-task-scheduling) to finish setting up your environment.
+
+## Red Hat OpenShift AI
+
+This Quick Start guide assumes that you have administrator access to an OpenShift cluster and an existing Red Hat OpenShift AI (RHOAI) installation with version **>2.9** is present on your cluster. But the quick step to install RHOAI is as follows:
+
+1. Using the OpenShift web console, navigate to **Operators** -> **OperatorHub**.
+2. Search for `Red Hat OpenShift AI`.
+3. Install it using the `fast` channel.
+
+### Configure Data Science Components
+
+After the installation of the Red Hat OpenShift AI Operator, proceed to configure the necessary components for data science work:
+
+1. From the OpenShift web console, navigate to the installed RHOAI Operator.
+2. Look for tab labeled DSC Initialization.
```

Comment:
I think that the DSCI is created while RHOAI is being installed and doesn't need to be created manually. Maybe double check this and have it as optional here - if it has not already been created.